src/nile/utils/debug.py

```diff
@@ -87,6 +64,9 @@ def _abi_to_build_path(filename):
 
 def _locate_error_lines_with_abis(file, addresses, to_contract):
     contracts = []
+    if not os.path.isfile(file):
+        raise IOError(f"Contracts file {file} not found.")
```

Comment:
in what context can this happen?

Reply:
If a user accidentally deletes/renames the deployments.txt file that's being used for the debug query.